<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>822</width>
    <height>495</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QFrame" name="frame">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(53, 53, 53);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QWidget" name="horizontalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>80</y>
         <width>451</width>
         <height>80</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="pushButton_4">
          <property name="text">
           <string>Selecionar arquivo...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="placeholderText">
           <string>Nenhum arquivo selecionado</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>210</y>
         <width>221</width>
         <height>80</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Id da area</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox">
          <property name="editable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>210</y>
         <width>221</width>
         <height>80</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Nome da area</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_2">
          <property name="editable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget_3">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>340</y>
         <width>221</width>
         <height>80</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Nome da região</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_3">
          <property name="editable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>20</y>
         <width>201</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>1. Upload map (shapefile)</string>
       </property>
      </widget>
      <widget class="QFrame" name="frame_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>810</width>
         <height>481</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>10</y>
          <width>151</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>2. Upload data (.csv)</string>
        </property>
       </widget>
       <widget class="QWidget" name="horizontalLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>40</y>
          <width>411</width>
          <height>80</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="pushButton_5">
           <property name="text">
            <string>Selecionar arquivo</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_2">
           <property name="placeholderText">
            <string>Nenhum arquivo selecionado</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget_4">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>160</y>
          <width>201</width>
          <height>81</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Id da área</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_4">
           <property name="editable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget_6">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>270</y>
          <width>201</width>
          <height>80</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>População</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_5">
           <property name="editable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget_7">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>270</y>
          <width>191</width>
          <height>80</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Casos</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_6">
           <property name="editable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget_8">
        <property name="geometry">
         <rect>
          <x>490</x>
          <y>40</y>
          <width>291</width>
          <height>80</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Tipo de covariavel</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QRadioButton" name="radioButton">
             <property name="text">
              <string>Invidivual</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButton_2">
             <property name="text">
              <string>Area</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget_5">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>160</y>
          <width>191</width>
          <height>78</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Data</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_7">
           <property name="editable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget_9">
        <property name="geometry">
         <rect>
          <x>480</x>
          <y>160</y>
          <width>141</width>
          <height>80</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_9">
         <item>
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Covariavel 1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_8">
           <property name="editable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget_10">
        <property name="geometry">
         <rect>
          <x>650</x>
          <y>160</y>
          <width>131</width>
          <height>80</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_10">
         <item>
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>Covariavel 2</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_9">
           <property name="editable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget_11">
        <property name="geometry">
         <rect>
          <x>650</x>
          <y>270</y>
          <width>131</width>
          <height>80</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_11">
         <item>
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>Covariavel 4</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_10">
           <property name="editable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget_12">
        <property name="geometry">
         <rect>
          <x>480</x>
          <y>270</y>
          <width>141</width>
          <height>80</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_12">
         <item>
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>Covariavel 3</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_11">
           <property name="editable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QPushButton" name="f2_return_f3">
        <property name="geometry">
         <rect>
          <x>620</x>
          <y>380</y>
          <width>171</width>
          <height>36</height>
         </rect>
        </property>
        <property name="text">
         <string>3.Selecionar análises &gt;</string>
        </property>
       </widget>
       <widget class="QPushButton" name="f2_return_f1">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>380</y>
          <width>221</width>
          <height>36</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt; 1. Upload mapa (shapefile)</string>
        </property>
       </widget>
       <widget class="QFrame" name="frame_3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>811</width>
          <height>481</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QLabel" name="label_15">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>10</y>
           <width>151</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>3. Selecionar análises</string>
         </property>
        </widget>
        <widget class="QWidget" name="horizontalLayoutWidget_4">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>50</y>
           <width>641</width>
           <height>61</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="spacing">
           <number>20</number>
          </property>
          <property name="leftMargin">
           <number>30</number>
          </property>
          <property name="rightMargin">
           <number>30</number>
          </property>
          <item>
           <widget class="QLabel" name="label_16">
            <property name="text">
             <string>Unidade de tempo:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButton_3">
            <property name="text">
             <string>Ano (AAAA)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButton_4">
            <property name="text">
             <string>Mês (AAAA-MM)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButton_5">
            <property name="text">
             <string>Dias (AAAA-MM-DD)</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="verticalLayoutWidget_13">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>270</y>
           <width>237</width>
           <height>80</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_13">
          <item>
           <widget class="QLabel" name="label_17">
            <property name="text">
             <string>Tipo de análise</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QRadioButton" name="radioButton_6">
              <property name="text">
               <string>Espacial</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButton_7">
              <property name="text">
               <string>Espaço temporal</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="verticalLayoutWidget_14">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>150</y>
           <width>292</width>
           <height>80</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_14">
          <item>
           <widget class="QLabel" name="label_18">
            <property name="text">
             <string>Intervalo de tempo</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="label_19">
              <property name="text">
               <string>de:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDateEdit" name="dateEdit"/>
            </item>
            <item>
             <widget class="QLabel" name="label_20">
              <property name="text">
               <string>até:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDateEdit" name="dateEdit_2"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QPushButton" name="button_start_analysis">
         <property name="geometry">
          <rect>
           <x>470</x>
           <y>200</y>
           <width>181</width>
           <height>81</height>
          </rect>
         </property>
         <property name="text">
          <string>Inicializar Análises</string>
         </property>
        </widget>
        <widget class="QPushButton" name="f3_return_f1">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>400</y>
           <width>211</width>
           <height>36</height>
          </rect>
         </property>
         <property name="text">
          <string>&lt;&lt; 1. Upload mapa (shapefile)</string>
         </property>
        </widget>
        <widget class="QPushButton" name="f3_return_f2">
         <property name="geometry">
          <rect>
           <x>330</x>
           <y>400</y>
           <width>181</width>
           <height>36</height>
          </rect>
         </property>
         <property name="text">
          <string>&lt; 2.Upload data (.csv)</string>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QPushButton" name="f1_return_f2">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>380</y>
         <width>181</width>
         <height>36</height>
        </rect>
       </property>
       <property name="text">
        <string>2. Upload data (.csv) &gt;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="f1_return_f3">
       <property name="geometry">
        <rect>
         <x>590</x>
         <y>380</y>
         <width>181</width>
         <height>36</height>
        </rect>
       </property>
       <property name="text">
        <string>3. Selecionar análises &gt;&gt;</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
